### ISSUE TITLE
OMEdit parameter dialog is missing the edit button for parameter record missing 

### ISSUE THREAD

**Issue by sbaetzing** [OpenModelica/OpenModelica#10700]
Not only replaceable parameter records should have the edit button in the parameter dialog, so also non replaceable parameter of record types should have it.

Tested with nightly OMEdit from 11.05.2023

```modelica
package MinimalExample

model Testclass

replaceable parameter testrecord test1;
parameter testrecord test2;


equation

end Testclass;

record testrecord
parameter Real A = 1;
parameter Real B = 2;
end testrecord;

  model test
  Testclass testclass annotation(
      Placement(visible = true, transformation(origin = {-38, -24}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
  equation

  end test;

end MinimalExample;
```

**Comment by casella** [comment]
@adeas31 I guess this is an easy modification, is it?

**Comment by casella** [comment]
@adeas31 can you check if we can get this in 1.23.0?

**Comment by adeas31** [comment]
Fixed and ported to 1.23 #12326
